release/cloudbuild.yaml

```diff
@@ -1,122 +0,0 @@
-# Copyright 2022 The Sigstore Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-timeout: 3600s
-
-steps:
```

Comment:
If we remove this, will we just be using github actions to release? According to https://github.com/sigstore/public-good-instance/blob/main/playbooks/release.md, don't we need cloudbuild to push to the registries? Or should this be added only once we transfer over to sigstore (and then I'll create a ticket to track)?

Reply:
i would like to build in github actions and not use cloudbuild if dont needed, this for me sounds the case as well, it is a small and quick release.

we are doing this for policy-controller already and we use the workload identity to connect to the gcp project to push the image to there.

when we transfer this project, I will update the job to include that, for now only github registry :)

Reply:
Sounds perfect, thanks!